packages/git-semver-tags/test.js

```diff
@@ -90,29 +88,35 @@ describe('git-semver-tags', function () {
   })
 
   it('should be in reverse chronological order', function (done) {
-    writeFileSync('test4', '')
-    shell.exec('git add --all && git commit -m"Fourth commit"')
-    shell.exec('git tag v1.0.0')
+    // new method is too fast
+    setTimeout(function () {
```

Comment:
wdym to fast?

Reply:
closing for now because I can't get the tests to pass reliably.